MAUI/StepProgressBar/migration.md

```diff
@@ -0,0 +1,425 @@
+
+using Syncfusion.Maui.ProgressBar;
+...
+
+SfStepProgressBar stepProgressBar = new SfStepProgressBar();
+this.Content = stepProgressBar ;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+## Classes 
+
+<table>
+<tr>
+<th>Xamarin SfStepProgressBar</th>
+<th>.NET MAUI SfStepProgressBar</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[SfStepProgressBar]()'| markdownify }}</td>
+<td>{{'[SfStepProgressBar]()'| markdownify }}</td>
+<td>Represents a class which holds the step progressbar common properties in SfStepProgressBar.</td>
+</tr> 
+<tr>
+<td>{{'[StepView ]()' | markdownify }}</td>
+<td>{{'[StepProgressBarItem]()' | markdownify }}</td>
+<td>Represents a class which is used to customize the step progress bar item.</td>
+</tr> 
+<tr>
+<td>{{'[IStepStyle]()' | markdownify }}</td>
+<td>{{'[StepSettings]()' | markdownify }}</td>
+<td>Represents a class which is used to customize the style of the step view.</td>
+</tr>
+<tr>
+<td>{{'[StepTappedEventArgs ]()' | markdownify }}</td>
+<td>{{'[StepTappedEventArgs]()' | markdownify }}</td>
+<td>Represents a class which is used to hold the step tapped event arguments.</td>
+</tr> 
+<tr>
+<td>{{'[StatusChangedEventArgs ]()' | markdownify }}</td>
+<td>{{'[StepStatusChangedEventArgs]()' | markdownify }}</td>
+<td>Represents a class which is used to hold the status changed event arguments.</td>
+</tr>
+</table>
+
+## Properties 
+
+### SfStepProgressBar
+
+The following code example, explains how to initialize the properties of the `Xamarin SfStepProgressBar` and `.NET MAUI SfStepProgressBar` class.
+
+<table>
+<tr>
+<th>Xamarin SfStepProgressBar</th>
+<th>.NET MAUI SfStepProgressBar</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:progressBar="clr-namespace:Syncfusion.XForms.ProgressBar;assembly=Syncfusion.SfProgressBar.XForms">
+
+    <progressBar:SfStepProgressBar />
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.XForms.ProgressBar;
+...
+
+SfStepProgressBar sfStepProgressBar = new SfStepProgressBar ();
+this.Content = sfStepProgressBar ;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:progressBar="clr-namespace:Syncfusion.Maui.ProgressBar;assembly=Syncfusion.Maui.ProgressBar">
+
+    <progressBar:SfStepProgressBar />
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.ProgressBar;
+...
+
+SfStepProgressBar stepProgressBar = new SfStepProgressBar();
+this.Content = stepProgressBar ;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+<table>
+<tr>
+<th>Xamarin SfStepProgressBar</th>
+<th>.NET MAUI SfStepProgressBar</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>{{'[CompletedStepStyle]()'| markdownify }}</td>
+<td>{{'[CompletedStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the completed step style for the step progressbar.</td>
+</tr>
+<tr>
+<td>{{'[InProgressStepStyle]()'| markdownify }}</td>
```

Comment:
<img width="434" alt="Screenshot 2024-03-12 at 6 18 42 PM" src="https://github.com/syncfusion-content/maui-docs/assets/46158936/79504836-95d9-4959-8b6e-40f3380c5271">
Description mentioned as style

Reply:
Step status descriptions are changed to Completed step settings